fuel/datasets/cifar10.py

```diff
@@ -33,23 +34,12 @@ class CIFAR10(H5PYDataset):
     filename = 'cifar10.hdf5'
 
     def __init__(self, which_set, **kwargs):
-        if which_set not in ('train', 'test'):
-            raise ValueError("CIFAR10 only has a train and test set")
-
         kwargs.setdefault('load_in_memory', True)
-        self.which_set = which_set
-
-        super(CIFAR10, self).__init__(self.data_path, which_set,
-                                      **kwargs)
+        super(CIFAR10, self).__init__(self.data_path, which_set, **kwargs)
+        self.default_transformers += (
```

Comment:
Yes, I think it's completely reasonable, and in cases where it should be done 90% of the time, we can always just add it to the default transformers (and then for the 10% that doesn't want to use it they can apply the transformers manually). I think I'd much prefer the class attribute approach, keeps the dataset / transformer divide a bit clearer.